runtime/app.go

```diff
@@ -155,4 +156,16 @@ func (a *App) RegisterTendermintService(clientCtx client.Context) {
 	)
 }
 
+// FindStoreKey fetches a registered StoreKey from the App in linear time.
+//
+// NOTE: This should only be used in testing.
+func (a *App) FindStoreKey(storeKey string) storetypes.StoreKey {
```

Comment:
```suggestion
func (a *App) UnsafeFindStoreKey(storeKey string) storetypes.StoreKey {
```

One thought I had is that we could enable/disable this with a `insecure` flag on the runtime module config. But since this can't be exposed to modules anyway (runtime depends on all the other modules so no module can get a direct reference to `runtime.App`), it is maybe okay for now.

Reply:
Sounds reasonable to me